Src/ST-LIB_LOW/ErrorHandler/ErrorHandler.cpp

```diff
@@ -8,8 +8,17 @@
 #include "ErrorHandler/ErrorHandler.hpp"
 
 string ErrorHandlerModel::description = "";
+string ErrorHandlerModel::line = "Error-No-Line";
+string ErrorHandlerModel::func = "Error-No-Line";
+string ErrorHandlerModel::file = "Error-No-File";
 double ErrorHandlerModel::error_triggered = 0;
 
+void ErrorHandlerModel::GetMetaData(int line, const char * func, const char * file){
+		ErrorHandlerModel::line = to_string(line);
+		ErrorHandlerModel::func = std::string(func);
+		ErrorHandlerModel::file = std::string(file);
```

Comment:
```suggestion
		ErrorHandlerModel::file = string(file);
```

O ponemos std en todo, o en nada. Creo que mejor en nada, añade los using al hpp